pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -568,9 +568,20 @@ func testInboundListenerConfigWithSidecarWithoutServicesV13(t *testing.T, proxy
 		},
 	}
 	listeners := buildInboundListeners(p, proxy, sidecarConfig)
-	if expected := 0; len(listeners) != expected {
+	if expected := 1; len(listeners) != expected {
```

Comment:
Is this saying we can now generate valid listener config for UDS? Otherwise I don't understand why it was passing

Reply:
Before this PR,  if there was no service instance, we would not generate any listener even if  the Sidecar had an explicit ingressListener spec. Thats why expected was set to 0. With this PR,  we expect to see ingress listeners. If you look at the sidecar spec in this test, it has
```
Ingress: []*networking.IstioIngressListener{
				{
					Port: &networking.Port{
						Number:   8080,
						Protocol: "unknown",
						Name:     "uds",
					},
					Bind:            "1.1.1.1",
					DefaultEndpoint: "127.0.0.1:80",
				},
			},
```

Hence the number of listeners is 1